addons/fleet/views/fleet_vehicle_views.xml

```diff
@@ -415,18 +391,14 @@
         <field name="arch" type="xml">
             <kanban class="o_kanban_mobile">
                 <templates>
-                    <t t-name="kanban-box">
-                        <div t-attf-class="oe_kanban_global_click">
-                            <div>
-                                <strong>
-                                    <field name="vehicle_id"/>
-                                    <span class="float-end"><field name="date"/></span>
-                                </strong>
-                            </div>
-                            <div>
-                                <span><field name="driver_id"/></span>
-                                <span class="float-end"><field name="value"/> Km</span>
-                            </div>
+                    <t t-name="kanban-card">
+                        <div>
+                            <field class="fw-bold" name="vehicle_id"/>
+                            <field class="fw-bold float-end" name="date"/>
```

Comment:
ms-auto, not float-end ?

Reply:
for ms-auto i have used d-flex on the div